RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Live.swift

```diff
@@ -303,27 +315,31 @@ extension FactorSourcesClient: DependencyKey {
 				assert(request.signers.allSatisfy { $0.networkID == request.networkID })
 				return try await signingFactors(
 					for: request.signers,
-					from: getFactorSources().asIdentified(),
+					from: getFactorSources(),
 					signingPurpose: request.signingPurpose
 				)
 			},
 			updateLastUsed: { request in
 				_ = try await profileStore.updating { profile in
-					var factorSources = profile.factorSources
+					var identifiedFactorSources = profile.factorSources.asIdentified()
```

Comment:
this is the standard pattern introduced by this PR, whenever we mutate `accounts`, `personas`, `factorSources`, `authorizedDapps`, `profileNetworks` etc, we start by creating a mutable `IdentifiedArrayOf` and mutate it, then write it back.